scripts/Modules/mkw_classes.py

```diff
@@ -40,6 +41,40 @@ class vec3:
   y: float = 0.0
   z: float = 0.0
   
+  def norm(self):
+    return math.sqrt(self.x**2 + self.y**2 + self.z**2)
+
+@dataclass
+class timer:
+  min: int
+  sec: int
+  mil: float
```

Comment:
Milliseconds aren't floats in MKW, but this might be an intentional decision to use a float here. Is the rationale that we might be working at a level below milliseconds?

Reply:
In terms of the Exact Finish Time calculation, we track fractions of milliseconds so we need to store a float here

Reply:
Works for me, though since we're in the mkw_classes file, I would recommend labeling this in a more specific way (e.g. `ExactTimer`) to avoid confusion.